epic_kitchens/dataset/epic_dataset.py

```diff
@@ -90,17 +90,45 @@ def __repr__(self):
 class EpicVideoDataset(VideoDataset):
 
     def __init__(
-        self, gulp_path: Path, class_type: str, class_getter=None, with_metadata=False
+        self,
+        gulp_path: Path,
+        class_type: str,
+        *,
+        with_metadata: bool = False,
+        class_getter: Optional[Callable[[Dict[str, Any]], Any]] = None,
+        segment_filter: Optional[Callable[[VideoSegment], bool]] = None,
+        sample_transform: Optional[Callable[[List[PIL.Image.Image]], List[PIL.Image.Image]]] = None
     ) -> None:
-        super().__init__(_class_count[class_type])
+        """
+
+        Parameters
+        ----------
+        gulp_path
+            Path to gulp directory containing the gulped EPIC RGB or flow frames
+        class_type
+            One of verb, noun, verb+noun, determines what label the segment returns
+        class_getter
```

Comment:
Swap ordering of docs to match signature (with_metadata comes before in sig)

Reply:
Fixed in 52af03f